patientIdentifierType.md

```diff
@@ -0,0 +1,112 @@
+# PatientIdentifierType
+
+## Overview
+Administrators define what types of identifiers they will collect. These range from National ID numbers, to driver's license numbers, to per-hospital medical record numbers.
+
+### Properties on PatientIdentifierType:
+
+* format: a regular expression defining what the identifier text should contain.
+
+* formatDescription: An optional description of the regular expression (used to explain the requirements of the regular expression in terms a user would understand). For example, a regular expression like \d{4,8}could have a description like "Must be a number between 4 and 8 digits in length."
+
+* required: a true/false whether every patient MUST have this type
```

Comment:
Are we ending property descriptions with a period or not? We should be consistent. Personally, I tend not to put periods at the end of brief descriptions like this – only using them when a description requires multiple sentences, since the period doesn't really add any value in a bulleted series of short descriptions, IMHO.

Reply:
definitely, will do this